README.md

```diff
@@ -60,9 +63,39 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+yum -y install epel-release
+yum -y install \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python2-pyyaml \
+    python2-six \
+    python-gobject-base \
+    python-jsonschema \
+    python-setuptools \
+    python-ipaddress
+echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+echo -e "[main]\nno-auto-default=*\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+systemctl restart NetworkManager
+systemctl restart openvswitch
+```
+
+### Install nmstate from PyPi
+```shell
+yum -y install python2-pip
+pip uninstall nmstate; pip install nmstate
+```
+
+### Install nmstate from source
+Install system-wide:
 ```shell
-sudo pip install --upgrade .
+pip install --upgrade .
```

Comment:
`pip uninstall nmstate; pip install .` would be better here, too. IMHO

Reply:
Done